Dockerfile

```diff
@@ -49,9 +56,8 @@ COPY . .
 # If we're building from a git clone, ensure that .git is writeable
 RUN [ -d .git ] && chown -R hypothesis:hypothesis .git || :
 
-# Build frontend assets
-RUN npm install --production \
-  && NODE_ENV=production node_modules/.bin/gulp build
+# Copy frontend assets.
+COPY --from=build /build .
```

Comment:
On my system with Docker version 17.12.0-ce, build c97c6d6, this puts the contents of the `build` directory in the root of the Docker image, instead of in `./build`.

Running the resulting image with `make run-docker` results in a non-working web-service (visiting http://localhost:5000 produces an nginx gateway error) due to h's inability to read `build/manifest.json`. After changing this line to:

```dockerfile
COPY --from=build /build build
```

The resulting Docker image worked as expected.